### PR TITLE
feature: shortcuts to navigate between Notes and Editor panel

### PR DESCRIPTION
**Pull request by briacp**
Unfortunately, this does not make the panel appear if they're not already visible -- it only switch focus between panels. As JC suggested, we can make the panel tab blinks and/or add menubar information.

## Which ticket is resolved?

- Shortcut to switch between Notes and Editor panels
  * https://sourceforge.net/p/omegat/feature-requests/1630

### CODE REVIEW

**Comment by miurahr** [comment]
@briacp  please push your commit to source forge. Otherwise, your branch and PR will be removed **automatically** 
 by a mirror daemon that force  overwrite github mirror repository by source forge git repository.

ref https://github.com/omegat-org/omegat/pull/317#issuecomment-1304874453

File: src/org/omegat/gui/notes/NotesTextArea.java [hunk 1]
```diff
@@ -137,12 +135,12 @@ public void redo() {
...
-                Preferences.setPreference(Preferences.NOTIFY_NOTES, notify.isSelected());
-            }
-        });
+        notify.addActionListener(e -> Preferences.setPreference(Preferences.NOTIFY_NOTES, notify.isSelected()));
         menu.add(notify);
     }
+    
+    @Override
+    public void requestFocus() {
+    	scrollPane.getViewport().getView().requestFocusInWindow();
```
Comment (miurahr) [inline src/org/omegat/gui/notes/NotesTextArea.java:144]: We can detect Note pane is hidden or not using a change suggested.

```suggestion
        if (scrollPane.getDockKey().getLocation().equals(DockableState.Location.HIDDEN)) {
            // FIXME: UIDesignManager.getDesktop(this).setAutoHide(<dockable here>, false);
        }
    	scrollPane.getViewport().getView().requestFocusInWindow();
```

I don't know what is `<dockable here>` object. If you find it, we can restore note pane when requested.

Comment (miurahr) [inline src/org/omegat/gui/notes/NotesTextArea.java:144]: With enhancement #349, we can just add `requestVisible`

```suggestion
        requestVisible();
    	scrollPane.getViewport().getView().requestFocusInWindow();
```

Comment (miurahr) [reply src/org/omegat/gui/notes/NotesTextArea.java:144]:
```suggestion
        StaticUIUtils.requestVisible(scrollPane);
    	scrollPane.getViewport().getView().requestFocusInWindow();
```

Comment (miurahr) [reply src/org/omegat/gui/notes/NotesTextArea.java:144]: @briacp please try the change on your environment?

Comment (briacp) [reply src/org/omegat/gui/notes/NotesTextArea.java:144]: That looks good to me, the `requestVisible` call works nicely.

Comment (brandelune) [reply src/org/omegat/gui/notes/NotesTextArea.java:144]: What's the status for this feature?

Comment (miurahr) [reply src/org/omegat/gui/notes/NotesTextArea.java:144]: waiting @briacp  to add a feature to restore pane by shortcut when note is invisible.

**Pull request closed without merging**